storage/entities/account.js

```diff
@@ -325,22 +325,34 @@ class Account extends BaseEntity {
 	/**
 	 * Create account object
 	 *
-	 * @param {Object} data
-	 * @param {Object} [options]
-	 * @param {Object} tx - Transaction object
+	 * @param {Object|Array.<Object>} data
+	 * @param {Object} [_options]
+	 * @param {Object} [tx] - Transaction object
 	 * @return {*}
 	 */
 	create(data, _options, tx) {
-		const objectData = _.defaults(data, defaultCreateValues);
-		const createSet = this.getValuesSet(objectData);
-		const attributes = Object.keys(data)
+		let values;
+
+		if (Array.isArray(data)) {
+			values = data;
+		} else if (typeof data === 'object') {
+			values = [data];
+		}
+
+		values = values.map(v => _.defaults(v, defaultCreateValues));
+
+		// We assume that all accounts have same attributes
+		// and pick first account attributes as template
+		const attributes = Object.keys(values[0]);
```

Comment:
Why not use `this.fields` or `defaultCreateValues` as attributes?
It's clearer than `values[0]` and avoid invalid fields if data contain any.